core/src/main/java/aima/core/search/basic/support/BasicNode.java

```diff
@@ -59,7 +62,7 @@ public String toString() {
 		this(state, null, null, pathCost);
 	}
 
-	BasicNode(S state, Node<A, S> parent, A action, double pathCost) {
+	public BasicNode(S state, Node<A, S> parent, A action, double pathCost) {
```

Comment:
This should not be 'public' as the intent is to only allow nodes to be instantiated via a NodeFactory (in this case a BasicNodeFactory).